src/site/content/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Enabling Shared Array Buffer
```

Comment:
On Desktop it's more a migration. As the post is covering both, I think the title is misleading.
What about Action to be taken to use SABs?

Reply:
How about "How to enable shared array buffer"?